scripts/basic-test.sh

```diff
@@ -46,15 +46,16 @@ if [ "$DARWIN" = true ]; then
 fi
 
 # fake-factory doesn't have a correct universal wheel
-pip install --no-use-wheel .[fakefactory]
+pip install --no-use-wheel faker
```

Comment:
I get the same, and thought about changing it, but I kinda want to be better about one fix per PR.

IOU one additional PR after this?